examples/gnrc_border_router/README.md

```diff
@@ -8,6 +8,35 @@ and routes on the BR.
 
 The script `start_network.sh` enables a *ready-to-use* BR in only one command.
 
+## Uplink
+
+The border router will route packets between a 6Lo network (PAN) and a 'normal'
+IPv6 network (i.e. the Internet).
+
+This requires the border router to have two interfaces, a downstream, 6lo-interface
+and an IPv6 uplink.
```

Comment:
6LoWPAN *is* IPv6 over X (where X is an LLN technology), so this wording is a bit confusing.

Reply:
I hope it's less confusing now.